setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = httpx
+version = attr: src.get_version
+url = https://github.com/encode/httpx
+license = BSD
+description = The next generation HTTP client.
```

Comment:
This is duplicated from `httpx.__version__` because this key doesn't support `attr:` if people are ok with me adding importlib_resources I'll change it to a `file:`